pkg/skaffold/schema/latest/config.go

```diff
@@ -69,6 +69,8 @@ type Metadata struct {
 
 // Pipeline describes a Skaffold pipeline.
 type Pipeline struct {
+	ConfigName string
```

Comment:
we can't modify the `Pipeline` struct here directly since that'll break the `skaffold.yaml` config.

Reply:
Done! I removed the `ConfigName`.